pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -135,7 +136,10 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
             bufferPeriodDuration,
             owner
         )
-        ProtocolFeeCache(protocolFeeProvider)
+        ProtocolFeeCache(
+            protocolFeeProvider,
+            ProviderFeeIDs(ProtocolFeeType.SWAP, ProtocolFeeType.YIELD, params.aumFeeId)
```

Comment:
There should be a test in ManagedPoolSettings for this.

Reply:
Done in https://github.com/balancer-labs/balancer-v2-monorepo/pull/1833/commits/02e200eeb68232680847849207e0dc9c346ee526